src/executables/csql.c

```diff
@@ -1439,6 +1439,17 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
 	  fprintf (csql_Error_fp, "Auto trace isn't allowed in SA mode.\n");
 	}
       break;
+
+    case S_CMD_SINGLELINE:
+      if (!strcasecmp (argument, "on"))
+	{
+	  csql_arg->single_line_execution = true;
+	}
+      else if (!strcasecmp (argument, "off"))
+	{
+	  csql_arg->single_line_execution = false;
+	}
```

Comment:
When the csql mode is '**_interactive_**', you have to print out current SINGLELINE MODE,
for example
```
csql> ;single
SINGLELINE IS ON
csql> ;single off
SINGLELINE IS OFF
```

Reply:
Thank you. I will ask you review again after I will add the function which is the status of 'SingleLine' mode